lib/src/utils.test.ts

```diff
@@ -0,0 +1,19 @@
+import {pathToFileURL} from 'url';
+import {pathToUrlString} from './utils';
+
+describe('utils', () => {
+  describe('pathToUrlString', () => {
+    it('encode relative path like `pathToFileURL`', () => {
```

Comment:
Add a test for percent-encoding paths that are already percent-encoded, so we'll catch issues like the one below.

Reply:
Added.